packages/graphql/node.js

```diff
@@ -8,7 +8,7 @@ var common = require('./lib/common');
 var fragmentsFile = require('./lib/util').getFragmentsFile();
 
 exports.Context = exports.createContext = common.Context.extend({
-  bootstrap: function (enhancedComponent) {
+  prepareEnhancedElement: function (enhancedComponent) {
```

Comment:
Is `prepareEnhancedElement` really the way we want to advertise this method? The name invites to mutate the `enhancedElement` whereas it's (currently) used to do derive some data out of it.

How about something like `beforeRender` or `willRender`?

Reply:
Or return the mutated object (or newly created) and use it to render.